e2e/features/address-validation.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Address details are validated
+
+  Background:
+    Given I am signed in as a Legal Rep
+    And I create a new case
+    And I complete the Home Office reference page
+    And I complete the Basic details page
+
+  @create-case @alternate @RIA-732
+  Scenario: Legal rep indicates there is no fixed address for the appellant
+
+    When I select Yes for the Does the appellant have a fixed address? field
+    When I should not see the text We will use the address of your legal practice.
```

Comment:
Think this should be a `Then` for clarity

Reply:
Quite right